### PR TITLE
Bump dekorate 3.3.0 to 3.3.1

### PR DESCRIPTION
**Pull request by Sgitario**
Changes https://github.com/dekorateio/dekorate/releases/tag/3.3.1 
Required for https://github.com/quarkusio/quarkus/pull/30480

### CODE REVIEW

**Review by manusa — APPROVED** [review]
:raised_hands: Thx!

**Review by gastaldi — APPROVED** [review]
LGTM, but I'll let @metacosm and/or @iocanel give the final say

**Comment by metacosm** [comment]
LGTM, if I understand correctly this is needed for the Fabric8 update so I'm all for it! ;)

**Review by metacosm — APPROVED** [review]

**Pull request merged**
